### ISSUE TITLE
Ubuntu 20.04 - Cannot add directory or network share

### ISSUE THREAD

**Issue by wepe53** [JonMagon/KDiskMark#43]
Hi,
I would like to use kdiskmark to check a nfs network share.
I successfuly installed kdiskmark within Ubuntu from the software package manager. However I only see a number of (local) directories, many of them are greyed. No way to add a dedicated directory. If I click on "add directory" I am lost in the file explorer and cannot select a specific directory (maybe there is a trick?).

I also tried to install kdiskmark via snap - this works but now I only have 1 directory (/home/.....) to select from.

When I run "sudo kdiskmark" then I get permission errors ("cannot create directory /run/user/0 - permission denied") - realpath - no such file or directory!

Any idea how to solve this poroblem?

Thanks
Wolfgang

**Comment by JonMagon** [comment]
Did you connect snap interfaces?

**Comment by wepe53** [comment]
Hi,
I am not sure wether I understand your question. I am not a real Linux 
expert.

Initially (I believe/remember) I installed kdiskmark from the Ubuntu 
20.04 software store or maybe via apt get. Within the kdiskmark app I 
was not able to add a new directory. The drop down list showed 3 entries 
which I could select, the other entries were greyed out.
When clicking on "add directory" I was forwarded to a file explorer but 
could not select a directory (I could only open it).

I then removed kdiskmark and installed again via snap. Same behaviour 
however this time only 1 directory was offered ( something like 
/home/.../snap/.../49 ).

Using sudo with kdiskmark produced permission error on /run/user/0 as 
far as I remember.

Same behaviour on Cent OS 8.

I do hope that this helps!

Basically what I want to achieve is to use kdiskmark to test a network 
share e.g. mount a remote NFS share to lets say /mnt/remnfs and test 
against this directory .

Thanks for your support!

Viele Grüße

Wolfgang Pein

Am 14.01.2021 um 18:01 schrieb Dmitry Sidorov:

**Comment by rejuce** [comment]
have excatly the same issue on lubuntu 20.04. installed the snap ...but only /home/jk/kdiskmark shows up as selctable directoy
if I click add aditional locations, qt file dialog pops up but if i select any path, just nothing happens...open without selection a file does not work...wrong type of qt filedialog used?  

if i select a specific file, file dialog closes but in the cmbo box dropdown nothin new is added except the single path that was there from the start

**Comment by JonMagon** [comment]
Related to #41
